integration/testnet/network.go

```diff
@@ -681,6 +706,13 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 
 			nodeContainer.bindPort(hostPort, containerPort)
 
+			hostMetricsPort := testingdock.RandomPort(t)
+			containerMetricsPort := "8080/tcp"
+
+			nodeContainer.bindPort(hostMetricsPort, containerMetricsPort)
+			nodeContainer.Ports[ColNodeMetricsPort] = hostMetricsPort
+			net.AccessPorts[ColNodeMetricsPort] = hostMetricsPort
```

Comment:
Great addition to make metrics accessible here. 

At the moment this will allocate and bind a host port for metrics for each container, but we will only store/print the ports of one container / node role, and we don't necessarily know _which_ container this is, if we have multiple with the same role). It would be great to be able to see metrics for all containers. 

What do you think about storing the metrics ports separately, keyed by container name (we do this with the `AccessPortsByContainerName` field to have distinguishable GRPC ports for each AN). Then we store a port for every container and can print out the corresponding container name alongside each metrics port. 
```
type FlowNetwork struct {
    // ...
    MetricPortsByContainerName map[string]string
}
```

Reply:
Good catch https://github.com/onflow/flow-go/pull/1763/commits/e5f55d1eb315704b2954d82d3800c6695bbb1d8e